### PR TITLE
Readd SuperClosure

### PR DESCRIPTION
**Pull request by nickvergessen**
I though it was a requirement of php-parser before, but actually it's used by the command bus stuff.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,12 +5,10 @@
 	"config": {
 		"vendor-dir": ".",
 		"optimize-autoloader": true,
-		"classmap-authoritative": true,
-		"platform": {
```
Comment (rullzer) [inline composer.json]: Why did you kill that?

Comment (nickvergessen) [reply composer.json]: It was a test to get rid of the php56 polyfill by requiring 7.1
Then I read up the topic and noticed that this here is basically useless?

Comment (rullzer) [reply composer.json]: Ah. It is actually used by dependabot. This is setting out minimum supported version so they don't try to bump to agressively

Comment (nickvergessen) [reply composer.json]: Okay, let me re add it

Comment (nickvergessen) [reply composer.json]: Done and restored alphabetical order

**Pull request merged**